### PR TITLE
fix: sending image to chat

### PR DESCRIPTION
**Pull request by wcchain**
fixes https://github.com/fanse-work/fanse-backend/issues/1

### CODE REVIEW

File: app/Http/Controllers/Api/v1/MediaController.php [hunk 1]
```diff
@@ -20,7 +20,7 @@ class MediaController extends Controller
      */
     public function store(Request $request)
     {
-        $this->authorize('create', Post::class);
```
Comment (gyroflaw) [inline app/Http/Controllers/Api/v1/MediaController.php:23]: Why did you comment this section? @wcchain

Comment (wcchain) [reply app/Http/Controllers/Api/v1/MediaController.php:23]: It causes the error and is not needed either

Comment (gyroflaw) [reply app/Http/Controllers/Api/v1/MediaController.php:23]: I think the error is occurred on Post::class authentication, if we remove this line we have authentication issue on chat file uploading.

Comment (wcchain) [reply app/Http/Controllers/Api/v1/MediaController.php:23]: it's an api and authorized by the route middleware already

Comment (gyroflaw) [reply app/Http/Controllers/Api/v1/MediaController.php:23]: Ok, I will accept, this PR. Good job @wcchain

**Review by gyroflaw — APPROVED** [review]

**Pull request merged**